pkg/config/config.go

```diff
@@ -13,13 +13,14 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const COMPONENT_VERSION = "2.5.0"
+const COMPONENT_VERSION = "2.7.0"
```

Comment:
Note: set to 2.7.0 because I plan to cherry pick this into release-2.7.  Will change main to 2.8 after that.